pkgs/desktops/xfce4-13/default.nix

```diff
@@ -66,7 +66,8 @@ makeScope newScope (self: with self; {
     inherit (gnome3) libsoup;
   };
 
-  xfce4-taskmanager = callPackage ./xfce4-taskmanager { };
+  xfce4-session = callPackage ./xfce4-session { };
+  xinitrc = "${xfce4-session}/etc/xdg/xfce4/xinitrc";
```

Comment:
What is this needed for?

Reply:
For https://github.com/nixos/nixpkgs/blob/88939a1/nixos/modules/services/x11/desktop-managers/xfce.nix#L125
The solution might be questionable, but if you want to refactor it will change ```xfce4_12``` interface too